packages/overlay/src/overlay.ts

```diff
@@ -65,6 +65,17 @@ export class Overlay {
         options: OverlayOptions
     ): Promise<() => void> {
         const overlay = new Overlay(owner, interaction, overlayElement);
+        if (window.__swc.DEBUG) {
+            window.__swc.warn(
+                undefined,
+                'As a direct consumer of the Overlay API, the SWC team wants to hear from you! Read the RFC below and share your thoughts on the future of overlays in the library.',
```

Comment:
Could we make it clear that action does (or doesn't) need to be taken by the developer? For example:

```suggestion
                'Overlay.open() is deprecated and may be removed in a future release. The SWC team wants feedback from direct Overlay API consumers like you - please share your thoughts on the future of overlays here:',
```

Reply:
Good call! Edited to `The Overlay API is currently being refactored and breaking changes, deprecations and/or removals are likely in a future release. The SWC team wants feedback from direct Overlay API consumers like you - please share your thoughts on the future of overlays here:`.

Working a canary release now, and assuming all goes well will merge later today.